.idea/codeStyleSettings.xml

```diff
@@ -135,9 +138,8 @@
           <option name="PARENT_SETTINGS_INSTALLED" value="true" />
           <indentOptions>
             <option name="INDENT_SIZE" value="2" />
-            <option name="CONTINUATION_INDENT_SIZE" value="2" />
+            <option name="CONTINUATION_INDENT_SIZE" value="4" />
```

Comment:
@tmorgner this has value="2" in [pentaho-coding-standards](https://github.com/pentaho/pentaho-coding-standards/blob/master/codeStyleSettings.xml#L89)